Makefile

```diff
@@ -93,7 +93,28 @@ kpod: .gopathok $(shell hack/find-godeps.sh $(GOPKGDIR) cmd/kpod $(PROJECT))
 crio.conf: crio
 	./crio --config="" config --default > crio.conf
 
-clean:
+tarball-prep:
+	git archive --format=tar.gz --prefix=cri-o-testonly/ HEAD > ../cri-o-testonly_1.0.0-rc2.orig.tar.gz
+
+test-rpm: tarball-prep
```

Comment:
Going by the example that we're using for integration tests, renaming this target to something like `local-test-rpm` and then having a new `test-rpm` target run something like `docker run -e TESTFLAGS -e TRAVIS -t --privileged --rm -v ${CURDIR}:/go/src/github.com/kubernetes-incubator/cri-o fedora:26 bash -c 'dnf -y install make git golang sudo rpm-build yum-utils; cd /go/src/github.com/kubernetes-incubator/cri-o; make local-test-rpm'` would be better for ensuring that the build environment for the test RPM package matches the target distribution.

Reply:
> the build environment for the test RPM package matches the target distribution

Modularity is suppose to fix that...but...yeah, environment sanity is important (I agree with you in principal).  Though, I'm not sure about depending on docker, to build the replacement for docker is a great idea :D  I think we can do that in spirit though, maybe using Ansible?  In any case, I think that could be left for another PR.  Getting it to work at all (esp. in automation) has to be the first-step, then we can enhance it under the guidance of CI feedback.  Just my $0.02

Reply:
I'm not enthusiastic about depending on a different container runtime to test cri-o, either, but we already do so for running the integration tests.

Doing everything in a `bash -c` invocation using a stock image was a for-example, though, and I'm not wedded to that detail.  The integration tests build a custom image to provide the build/test environment, which has to be easier to troubleshoot.

Reply:
Ya, the trouble is twofold: docker breaks or has a bug or we switch to kpod and get into the weeds ala docker-in-docker.  What about using simpler technology, like a chroot "jail"?  Not sexy to be sure, but it's more about the packaging-piece than the strict-isolation isn't it?  Anyway, it's a bit of a perpendicular topic, but certainly a conversation worth having.

I think what we have here meets the 80%/80% rule.  We can both setup and verify the environment from Ansible, then when/if Modularity rolls around, we have an easy path forward into that (i.e. since build is already orchestrated from Ansible, it can just use a different set of commands).

Reply:
I haven't changed anything here yet in my latest force-push, but I'm open to changing it either way that we agree upon.